Fix installation of the printer-driver-cnijfilter meta-package on amd64

mariospr opened the pull request:
Many (75) of the packages that are listed as dependencies of this meta-package
are 'Architecture: i386' only, and that is blocking the installation of the
whole suite of drivers on amd64, even if 58 packages in the list would still
be suitable for 64bit systems.

To fix this problem, let's move all the i386-only packages to the list of
'Recommends' instead of 'Depends' so that they will only be installed when
available, without blocking the installation of the whole suite.

https://phabricator.endlessm.com/T12338

dbnicholson commented on debian/control:
```diff
@@ -946,142 +946,150 @@ Description: IJ Printer Driver for Linux.
+	cnijfilter-mx890series,
+	cnijfilter-mx920series,
+	cnijfilter-p200series
+Recommends: printer-driver-cnijfilter-32
```
I forgot how this crazy package puts things together, but I think it would be best to just mark the i386 only drivers with `[i386]` rather than using Recommends.

Commit:
Fix installation of the printer-driver-cnijfilter meta-package on amd64

Many (75) of the packages that are listed as dependencies of this meta-package
are 'Architecture: i386' only, and that is blocking the installation of the
whole suite of drivers on amd64, even if 58 packages in the list would still
be suitable for 64bit systems.

To fix this problem, let's mark the i386-only packages as [i386] in the list of
dependencies for the meta-package so that they won't block the installation of
the whole suite on amd64.

https://phabricator.endlessm.com/T12338
